students/stuartkershaw/lesson07/test_html_render.py

```diff
@@ -0,0 +1,211 @@
+import pytest
+from io import StringIO
+
+from html_render import Element
+from html_render import Html
+from html_render import Body
+from html_render import P
+from html_render import Head
+from html_render import Title
+from html_render import Hr
+from html_render import A
+from html_render import H
+from html_render import Meta
+
+
+# Step 1
+
+
+def test_type_Element():
+    test_element = Element(content=None)
+    assert isinstance(test_element, Element) is True
+
+
+def test_class_attributes_Element():
+    test_element = Element(content=None)
```

Comment:
`content` default is None, no need to explicitly create `Element` with that default